include/freetds/tds.h

```diff
@@ -1324,6 +1324,21 @@ TDSICONV *tds_iconv_from_collate(TDSCONNECTION * conn, TDS_UCHAR collate[5]);
 
 
 /* mem.c */
+typedef struct tds_tvp_row
+{
+	TDSPARAMINFO * params;
+	struct tds_tvp_row * next;
+} TDS_TVP_ROW;
+
+typedef struct tds_tvp
+{
+	char * schema;
+	char * typename;
```

Comment:
Seems that you only changed some of the `typename`, please do a clean up for it.